extensions/wikia/MarkWikiAsClosed/controllers/MarkWikiAsClosedController.class.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Wikia\Logger\Loggable;
+
+class MarkWikiAsClosedController extends WikiaController {
+
+    use Loggable;
+
+    const WIKI_ID = 'wikiId';
+    const REASON = 'reason';
+
+    public function init() {
+        $this->response->setFormat(WikiaResponse::FORMAT_JSON);
+    }
+
+    public function markWikiAsClosed() {
+
+        $wikiId = $this->request->getVal(self::WIKI_ID);
```

Comment:
since you plan to put wikiId as query param, do you plan to restrict this api to community.wikia.com? or will it be available on every wiki?